.rubocop.yml

```diff
@@ -7,6 +7,10 @@ Style/FrozenStringLiteralComment:
 Style/ModuleFunction:
   EnforcedStyle: extend_self
 
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    '%i': ()
```

Comment:
The default _was_ `()` (this issue has only just started firing) and it still suggests that `%w()` is correct. Annoyingly the behaviour actually seems to be _different_ locally vs on CodeClimate and this way forces them to be the same. I'll change to force it to `[]`.